corehq/apps/hqwebapp/session_details_endpoint/views.py

```diff
@@ -68,7 +68,7 @@ def post(self, request, *args, **kwargs):
             domains.add(member_domain)
             domains.update(EnterprisePermissions.get_domains(member_domain))
 
-        enabled_toggles = toggles_enabled_for_user(user.username) | toggles_enabled_for_domain(domain)
+        enabled_toggles = toggles_enabled_for_request(request)
```

Comment:
I don't think you can pass in `request` here since it doesn't have a `domain` or `user` attribute